### ISSUE TITLE
Create  breadcrumb component

### ISSUE THREAD

**Issue by nishitapande** [purnima143/Pluto-UI#14]

**Comment by Lovelyfin00** [comment]
Hey @nishitapande 
Are you still interested in this?

**Comment by nishitapande** [comment]
@Lovelyfin00 hey. yes actually, I am interested in working on this. but this is not assigned to me. @purnima143 if you could assign this

**Comment by Lovelyfin00** [comment]
We don't assign issues but the first to indicate interest is given 3 days to work and create a PR for it

**Comment by Lovelyfin00** [comment]
So I'm going to refactor the description and then you can start working on it. I'll also include the link to the design for the breadcrumbs

**Comment by nishitapande** [comment]
oh. Thanks for including the design as well. Just another question are you the mentor GSSoc, cause like if I have some doubts/ issues, who can help me? If you could confirm this

**Comment by Lovelyfin00** [comment]
> oh. Thanks for including the design as well. Just another question are you the mentor GSSoc, cause like if I have some doubts/ issues, who can help me? If you could confirm this

Yes I am one of the mentors for this project. You can join the Pluto Ui channel and tag me @Lovelyfin00 if you have any quesxtions

**Comment by nishitapande** [comment]
can you please share the link for the channel? thanks

**Comment by Lovelyfin00** [comment]
Go to self role and select Pluto UI from the list of projects there
Then a new channel; would be added called pluto UI. You can join there

**Comment by nishitapande** [comment]
got it thanks

**Comment by tripathishipra** [comment]
Hi , I am contributor under GSSOC'23 and I am eager to showcase my skills and make a meaningful contribution to this project.
Kindly assign me this issue.@nishitapande

**Comment by nishitapande** [comment]
@tripathishipra hi there. I am also a contributor. and currently, this issue is assigned to me. but you can ask @Lovelyfin00 to assign you another issue.
